### PR TITLE
Switch to YAML and KDE Sdk 5.11

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
bot, build

**Comment by mtmiller** [comment]
You will need to apply [cdaa884568b1](https://hg.savannah.gnu.org/hgweb/octave/rev/cdaa884568b1) against 4.4 to compile with Qt 5.11.

**Comment by Alexander-Wilms** [comment]
Thanks.

bot, build

File: qt511.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- a/libgui/src/settings-dialog.cc
```
Comment (mtmiller) [inline qt511.patch]: I think it's helpful to include the complete hg metadata in the patch file.

Comment (Alexander-Wilms) [reply qt511.patch]: Can you do that? I just copied the patch from hgweb.

Comment (mtmiller) [reply qt511.patch]: Done in https://github.com/Alexander-Wilms/org.octave.Octave/pull/1.

**Pull request merged**